app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -66,74 +66,91 @@ describe("the new project page", () => {
     // MILESTONE REPORTS
     cy.findByText(/Milestone reports/i).click();
     cy.findByText(/Add milestone reports/i).click();
-    cy.findByText(/Add another milestone report/i).click();
```

Comment:
Removed addition of reports in this test because that's already covered by the `create-project-revision` tests